src/fmu/dataio/_models/fmu_results/fields.py

```diff
@@ -215,11 +187,11 @@ class Case(BaseModel):
 class Ert(BaseModel):
```

Comment:
the `fmu.ert` block is still **optional.** This change just requires the fields below if `fmu.ert` is populated.